src/zc/FileStorage/tests.py

```diff
@@ -23,14 +23,15 @@
 import ZODB.tests.testblob
 import doctest
 import pickle
```

Comment:
Maybe we should use zodbpickle instead?

Reply:
I think that will be a necessity if zc.FileStorage ever wants to support Python 3, like the regular ZODB.FileStorage does.